sys/linux/dev_mali.txt

```diff
@@ -331,6 +335,17 @@ kbase_ioctl_context_priority_check {
 	priority	flags[queue_group_priority, int8]
 }
 
+prfcnt_control_cmd {
+	cmd		flags[control_cmd, int8]
```

Comment:
This must be int16:

```
struct prfcnt_control_cmd {
        __u16 cmd;
        __u16 pad[3];
        __u64 user_data;
};
```

Reply:
I have addressed the points you mentioned! Thank you so much for your valuable feedback!